testdata/scripts/make_well_known_depth_sample_axis.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+from make_vds import *
```

Comment:
Please specify what functions and classes that are used.
I think it is better to import segyio directly, instead of importing it via 'make_vds'